xds/internal/httpfilter/fault/fault_test.go

```diff
@@ -506,7 +508,7 @@ func (s) TestFaultInjection_Unary(t *testing.T) {
 			}
 
 			// Create a ClientConn and run the test case.
-			cc, err := grpc.Dial("xds:///myservice", grpc.WithTransportCredentials(insecure.NewCredentials()))
+			cc, err := grpc.Dial("xds:///"+serviceName, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
Given how we maintain the refCount on the `xdsClient` it would be better to use `grpc.DialContext(..... grpc.WithBlock())` here.

Reply:
Why?  I don't understand.